sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/LockRenewalOperation.java

```diff
@@ -183,14 +183,16 @@ public void close() {
         final OffsetDateTime now = OffsetDateTime.now();
         Duration initialInterval = Duration.between(now, initialLockedUntil);
 
-        if (initialInterval.isNegative()) {
-            logger.info("Duration was negative. now[{}] lockedUntil[{}]", now, initialLockedUntil);
+        if (initialInterval.isNegative() || initialInterval.toMillis() < 400) {
```

Comment:
Is this a service-enforced restriction to have the initial interval be at least 400 ms? We should add a comment specifying why we picked 400 ms. Also, we should update the log message with this information about why less than 400ms is not supported.

Reply:
This is just so that we have the same renewal behaviour as the C# sdk, which I believe was inhertied from track 1, will update the change log adding this update and drop in a comment as well so that this magic number is explained 😄